radio/src/targets/horus/csd203_sensor.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) EdgeTX
+ *
+ * Based on code named
+ *   opentx - https://github.com/opentx/opentx
+ *   th9x - http://code.google.com/p/th9x
+ *   er9x - http://code.google.com/p/er9x
+ *   gruvin9x - http://code.google.com/p/gruvin9x
+ *
+ * License GPLv2: http://www.gnu.org/licenses/gpl-2.0.html
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#ifndef _CURRENTSENSOR_H_
+#define _CURRENTSENSOR_H_
+
+//#pragma once
```

Comment:
I think we are preferring `#pragma once` include guards now

Reply:
This driver may need more work in this branch.  Because it shares the same I2C with touch screen, not sure what is the best way to code I2C sharing for stability.